util/crs-rules-check/examples/test8.conf

```diff
@@ -4,23 +4,23 @@ SecRule &TX:blocking_paranoia_level "@eq 0" \
     phase:1,\
     pass,\
     nolog,\
-    ver:'OWASP_CRS/4.0.0-rc1',\
```

Comment:
These are probably unintended changes.

Reply:
Yeah, but in the end it always pops up, so 🤷

Reply:
Probably need a way to exclude directories from the update.